### DISCUSSION TITLE
Install Error

### DISCUSSION THREAD

**Discussion by halmerialk** [pimcore/pimcore#11747] [Q&A / Support] [answered]
I am trying to install pimpcore into my laptop (MAMP running)
After some valid steps (steps 1, 2, 3 in the installation guide), i try to install the software using (step 4. launch installer)
./vendor/bin/pimcore-install
And after giving the user, password, database etc an error is shown:

[ERROR] The following errors were encountered during installation                                                      
 * WITH RECURSIVE is required.
This is the error log in dev.log
[2022-03-23T15:18:43.880315+00:00] console.DEBUG: Command "pimcore:install" exited with code "2" {"command":"pimcore:install","code":2} []

Could anyone help me to install this software?
![Captura de pantalla 2022-03-23 a las 16 40 16](https://user-images.githubusercontent.com/91467302/159738174-1a5be13a-02fb-4af9-b01a-3d59d24f4ba5.png)

**Comment by jdreesen** [comment, marked as answer]
Are you sure you're using [MariaDB >= 10.3 or MySQL >= 8.0](https://pimcore.com/docs/pimcore/current/Development_Documentation/Installation_and_Upgrade/System_Requirements.html#page_Database-Server)?

**Reply by halmeriarsa** [reply]
Yest, it seems that pimcore was detecting an older version of mysql.
I have found the solution. 
It seems that i have multiple mysql versions running in my mac.
After some googling, i have found that you can set the mysql host this way:
sudo ./vendor/bin/pimcore-install --ignore-existing-config --mysql-host-socket=127.0.0.1
